src/index.html

```diff
@@ -1,14 +1,70 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="crossorigin">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+
+  <body class="body">
+    <header class="header">
+      <a href="/" class="header__logo">
+        <img src="images/logo.png" alt="MOYO logo"
+        class="header__logo__image">
```

Comment:
fix code style

Reply:
![image](https://github.com/mate-academy/layout_moyo-header/assets/106925415/f78c546f-e320-41e7-817f-6671e5e66dce)

What exactly am I supposed to fix?

Reply:
Don't forget to check the `checklist`, there are lots of useful tips and examples of good code style.
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/71a18726-8bf1-40f0-9b9e-52dee04d4ebc)

Reply:
Thank you! Done!